### DISCUSSION TITLE
"For you" is wholly unwelcome

### DISCUSSION THREAD

**Discussion by heliosyne** [community/community#65132] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

The "For You" tab adds no value. In fact, it reduces the site's value. It wastes my time by wresting from me the control I have with follows/stars. That algorithmic feed concept is the single greatest detractor of social media. It is an example of how not to do things, not something to emulate.

**Comment by darkwater4213** [comment]
100% agree. It should _never_ be the default. There's an argument to be made that it should have to be enabled in settings to even present the option. I don't personally agree with that, but it's there. The default landing page sholud always and forevermore be the Following page and _nothing else_. @heliosyne is dead-on... it's an example of what _not_ to do. I do enough doomscrolling as it is.

(How pathetic is it that I tend to doomscroll through neat programming projects?)

**Comment by ddurschlag** [comment]
github is not a social media site, and I don't want it tracking me or "recommending" anything. @heliosyne has it exactly right. This is not a feature, it's a bug.
